### ISSUE TITLE
How to remove field from child form?

### ISSUE THREAD

**Issue by ghprod** [kristijanhusak/laravel-form-builder#183]
Hi

usually to remove field from rendering we can use

```php
$form->remove('fieldname');
```

but how to remove field from child form?

```php
$this
            ->add('main', 'form', [
                'class' => MainForm::class,
                'label'  => false,
            ])
```

Where `MainForm::class` consists a lot of fields

I've try about `$form->remove('main.fieldname-in-child')` but still no luck .. since i saw that `remove` function just check for array key exists

Thanks

**Comment by kristijanhusak** [comment]
Hi,

I think something like this should work:

```php
$form->main->remove('fieldname');
```

Haven't tried it though.

**Comment by ghprod** [comment]
Hi

```php
dd($form->main->getForm()->remove('fieldname'));
```

With above code i can see that's success removed, but when rendered is still shown, any idea?

Thanks

**Comment by kristijanhusak** [comment]
Yeah, it wasn't working properly. I fixed it. Pull in `dev-master`, and use `removeChild` method like this:

```php
$form->main->removeChild('fieldname');
```

Let me know if it's working fine.

**Comment by ghprod** [comment]
Great @kristijanhusak .. :+1: 

its works now using latest commit

```
"kris/laravel-form-builder": "dev-master#5454300"
```

I hope you can soon release it to 1.6 :)

Thanks a lot :+1:

**Comment by kristijanhusak** [comment]
@ghprod 
I just released it into 1.6.31 :)

**Comment by ghprod** [comment]
Thanks a lot :+1: 

Super !!

![image](https://cloud.githubusercontent.com/assets/1922652/11183331/b2067a06-8ca3-11e5-8a17-64609294a283.png)
